Orange/preprocess/preprocess.py

```diff
@@ -10,7 +10,7 @@
 from . import impute, discretize
 from ..misc.enum import Enum
 
-__all__ = ["Continuize", "Discretize", "Impute", "SklImpute"]
+__all__ = ["Preprocess", "Continuize", "Discretize", "Impute", "SklImpute"]
```

Comment:
It was decided that __all__ should contain only classes that a user should see (and would use).
If other classes are needed elsewhere in the code they should be imported explicitly with the full path.